babel.config.js

```diff
@@ -14,30 +14,46 @@
  * limitations under the License.
  */
 
-/**
- * WordPress dependencies
- */
-const defaultConfig = require('@wordpress/babel-preset-default');
-
 module.exports = function (api) {
-  const config = defaultConfig(api);
+  const isTest = api.env('test');
+  const isProduction = api.env() === 'production';
```

Comment:
Should this be `api.env('production')` for consistency?

Reply:
Makes sense. Updated now.